molecule/testinfra/staging/app/iptables-app-staging.j2

```diff
@@ -4,8 +4,8 @@
 :OUTPUT DROP
```

Comment:
Same point as above for the prod iptables template.

Reply:
:+1: